tests/component/test_stream_readers_ai.py

```diff
@@ -167,9 +167,10 @@ def test___analog_single_channel_reader___read_waveform_feature_disabled___raise
     ai_single_channel_task_with_timing: nidaqmx.Task,
 ) -> None:
     reader = AnalogSingleChannelReader(ai_single_channel_task_with_timing.in_stream)
+    waveform = AnalogWaveform[numpy.float64](50)
```

Comment:
https://github.com/ni/nidaqmx-python/pull/807#discussion_r2283511149

> Just pass the sample count. AnalogWaveform will allocate that many samples. The default dtype is np.float64.

Please remember feedback from previous PRs and continue to apply it.